cmd/subnetcmd/upgradecmd/print_upgrade.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet upgrade import
+func newUpgradePrintCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print [subnetName]",
+		Short: "Print the upgrade.json file content",
+		Long:  `Print the upgrade.json file content`,
+		RunE:  upgradePrintCmd,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	return cmd
+}
+
+func upgradePrintCmd(cmd *cobra.Command, args []string) error {
```

Comment:
One question I have, should we be supporting one set of upgrade bytes per network? Should we be able to have one set of upgrade bytes for fuji, one for local, and one for mainnet? Probably best to address that in your first PR or a follow up, but if you did add that it would require a flag/prompt here to pick a network.

Reply:
Hmmm good question. We probably should, although I think the user is doing herself a disservice if she's keeping different versions for the same subnet - can quickly become pretty messy (in the sense that she really needs to know what she's doing).

If we do that, then all the previous cmds need this flag.

Reply:
Let's address this in another ticket. My thinking is that you want to test any updates you make locally and then on Fuji before you push them to mainnet. Kind of like a pipeline.

Reply:
Created #372 for this.